### ISSUE TITLE
Nuke crashes server

### ISSUE THREAD

**Issue by xskutsu** [Jboymercs/end_expansion_final#11]
When the nuke is triggered on a dedicated server, it crashes the game.

```
net.minecraft.util.ReportedException: Ticking entity
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:773) ~[MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.updateTimeLightAndEntities(DedicatedServer.java:397) ~[nz.class:?]
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668) ~[MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:1817) ~[MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:1570) [?:?]
Caused by: java.lang.NoClassDefFoundError: com/example/structure/event_handler/ClientRender
        at com.example.structure.entity.endking.EntityNuclearExplosion.onUpdate(EntityNuclearExplosion.java:173) ~[EntityNuclearExplosion.class:?]
        at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1996) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntityWithOptionalForce(WorldServer.java:832) ~[oo.class:?]
        at net.minecraft.world.World.updateEntity(World.java:1958) ~[amu.class:?]
        at net.minecraft.world.World.updateEntities(World.java:1762) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613) ~[oo.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767) ~[MinecraftServer.class:?]
        ... 4 more
Caused by: java.lang.ClassNotFoundException: com.example.structure.event_handler.ClientRender
        at top.outlands.foundation.boot.ActualClassLoader.findClass(ActualClassLoader.java:260) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.loadClass(ActualClassLoader.java:267) ~[foundation-0.14.0.jar:?]
        at com.example.structure.entity.endking.EntityNuclearExplosion.onUpdate(EntityNuclearExplosion.java:173) ~[EntityNuclearExplosion.class:?]
        at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1996) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntityWithOptionalForce(WorldServer.java:832) ~[oo.class:?]
        at net.minecraft.world.World.updateEntity(World.java:1958) ~[amu.class:?]
        at net.minecraft.world.World.updateEntities(World.java:1762) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613) ~[oo.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767) ~[MinecraftServer.class:?]
        ... 4 more
Caused by: net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerException: Exception in class transformer net.minecraftforge.fml.common.asm.transformers.SideTransformer@2d4608a6 from coremod FMLCorePlugin
        at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:261) ~[cleanroom-0.2.4-alpha.jar:?]
        at top.outlands.foundation.TransformerDelegate.lambda$fillTransformerHolder$1(TransformerDelegate.java:150) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.runTransformers(ActualClassLoader.java:332) ~[foundation-0.14.0.jar:?]
        at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:68) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.findClass(ActualClassLoader.java:244) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.loadClass(ActualClassLoader.java:267) ~[foundation-0.14.0.jar:?]
        at com.example.structure.entity.endking.EntityNuclearExplosion.onUpdate(EntityNuclearExplosion.java:173) ~[EntityNuclearExplosion.class:?]
        at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1996) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntityWithOptionalForce(WorldServer.java:832) ~[oo.class:?]
        at net.minecraft.world.World.updateEntity(World.java:1958) ~[amu.class:?]
        at net.minecraft.world.World.updateEntities(World.java:1762) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613) ~[oo.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767) ~[MinecraftServer.class:?]
        ... 4 more
Caused by: java.lang.RuntimeException: Attempted to load class com/example/structure/event_handler/ClientRender for invalid side SERVER
        at net.minecraftforge.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:62) ~[cleanroom-0.2.4-alpha.jar:?]
        at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:257) ~[cleanroom-0.2.4-alpha.jar:?]
        at top.outlands.foundation.TransformerDelegate.lambda$fillTransformerHolder$1(TransformerDelegate.java:150) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.runTransformers(ActualClassLoader.java:332) ~[foundation-0.14.0.jar:?]
        at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:68) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.findClass(ActualClassLoader.java:244) ~[foundation-0.14.0.jar:?]
        at top.outlands.foundation.boot.ActualClassLoader.loadClass(ActualClassLoader.java:267) ~[foundation-0.14.0.jar:?]
        at com.example.structure.entity.endking.EntityNuclearExplosion.onUpdate(EntityNuclearExplosion.java:173) ~[EntityNuclearExplosion.class:?]
        at net.minecraft.world.World.updateEntityWithOptionalForce(World.java:1996) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntityWithOptionalForce(WorldServer.java:832) ~[oo.class:?]
        at net.minecraft.world.World.updateEntity(World.java:1958) ~[amu.class:?]
        at net.minecraft.world.World.updateEntities(World.java:1762) ~[amu.class:?]
        at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613) ~[oo.class:?]
        at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767) ~[MinecraftServer.class:?]
        ```

**Comment by Jboymercs** [comment]
Yeah, this is on our bug list. I'd just disable its crafting recipe via craft tweaker atm until its fixed, which may be awhile as End Expansion it pending a massive update. But thank you nonetheless!

**Comment by Jboymercs** [comment]
Fixed in End Expansion Hotfix 1.2! Temporarily removed Screenshake from the mini-nukes blow off time.
